org.eclipse.jdt.core/model/org/eclipse/jdt/core/JavaCore.java

```diff
@@ -3323,7 +3323,7 @@ public static List<String> getAllVersions() {
 	 *
 	 * @return all Java source versions fully supported by Eclipse compiler
 	 * @see #isJavaSourceVersionSupportedByCompiler(String)
-	 * @since 3.39
+	 * @since 3.40
```

Comment:
Why is this required? This was added last release and we don't change the @since versions once they are set.